deploy/edgemax.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
please change the shebang to the same as the others'

Reply:
I'm confused on this request.  When I look at all of the other deploy scripts, they are just 
#!/bin/bash
Same as this one.
What do you want me to change?

Reply:
https://github.com/Neilpang/acme.sh/blob/master/deploy/vsftpd.sh#L1